.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDateParser.cs

```diff
@@ -893,6 +899,17 @@ private DateTimeResolutionResult Match2Date(Match match, DateObject referenceDat
             ret.PastValue = futurePastDates.past;
             ret.Success = true;
 
+            // Ambiguous two-digit years are assigned values in both centuries (e.g. 35 -> 1935, 2035)
+            if (ambiguousCentury)
+            {
+                ret.PastValue = futurePastDates.past.AddYears(1900);
```

Comment:
I'd prefer if 1900 and 2000 are constants in Constants.cs 
Maybe BASE_YEAR_PAST_CENTURY and BASE_YEAR_CURRENT_CENTURY?

Reply:
done